### PR TITLE
DRILL-7596: drill-format-esri module Uses Hard Coded Version Number

### PR DESCRIPTION
**Pull request by cgivre**
# [DRILL-7596](https://issues.apache.org/jira/browse/DRILL-7596): drill-format-esri module Uses Hard Coded Version Number

## Description

Removes hard coded version number from `format-esri`.

## Documentation
No user visible changes

## Testing
Ran unit tests associated with this format plugin.

### CODE REVIEW

File: contrib/format-esri/pom.xml [hunk 1]
```diff
@@ -72,7 +72,7 @@
   <dependency>
     <groupId>org.apache.drill.contrib</groupId>
     <artifactId>drill-udfs</artifactId>
```
Comment (vvysotskyi) [inline contrib/format-esri/pom.xml]: Is this dependency used in code, or it is required for tests only? If for tests, please specify the corresponding scope for it.
Also, please fix the alignment of `<dependencies>` block, it should be indented with two spaces.

Comment (cgivre) [reply contrib/format-esri/pom.xml]: @vvysotskyi 
Addressed review comments.  This dependency is only required for a specific unit test.

File: contrib/format-esri/pom.xml [hunk 2]
```diff
@@ -18,7 +18,8 @@
     limitations under the License.
 
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (vvysotskyi) [inline contrib/format-esri/pom.xml]: Please revert this change. `maven-release-plugin` automatically reformats the header line when `release:prepare` is performed.

Comment (cgivre) [reply contrib/format-esri/pom.xml]: Reverted first line spacing.  Just as a side note, I ran Drill's code formatting macro in Intellij which did this.  We might want to think about updating that so that this line is formatted correctly in future PRs, but that is out of scope for this PR.

Comment (vvysotskyi) [reply contrib/format-esri/pom.xml]: I agree that our templates for both IDEA and Eclipse are a little bit outdated, but I believe they will be updated when we will add more checkstyle rules and reformat the project for them ([DRILL-7352](https://issues.apache.org/jira/browse/DRILL-7352)).

But for now, sorry for asking to do additional changes.

Comment (cgivre) [reply contrib/format-esri/pom.xml]: No worries.  This kind of stuff is good for Drill. :-)

**Comment by arina-ielchiieva** [comment]
@cgivre please rebase and squash the commits.

**Comment by cgivre** [comment]
Rebased and commits squashed.  Thanks for the review.
